starter-code/index.html

```diff
@@ -1,44 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify</title>
+    <link rel="stylesheet" href="css\main.css">
   </head>
-  <body>
-    Premium Discover Help Download
-
-    Music for everyone.
-    Spotify is now free on mobile, tablet and computer.
-    Listen to the right music, wherever you are.
 
-    What’s on Spotify?
+  <body>
+    <div >
+       <nav class="navbar" >
+        <img class="imagelogo"  src="images\spotify-logo.png" alt="Logo" />
+        <ul class="linking">
+          <li><a href="#"></a>  Premium</li>
+          <li><a href="#"></a> Discover</li>
+          <li><a href="#"></a> Help</li>
+          <li><a href="#"></a> Download</li>
+        </ul>
+      </nav>
+    </div>
```

Comment:
Nice job! 
In this case I wouldn´t have wrapped a nav in the div as it wasnt necessary. :)